utility/src/test/java/org/oppia/util/data/DataProvidersTest.kt

```diff
@@ -2320,7 +2322,8 @@ class DataProvidersTest {
     val baseProvider1 = createSuccessfulDataProvider(BASE_PROVIDER_ID_0, STR_VALUE_0)
     val baseProvider2 = createSuccessfulDataProvider(BASE_PROVIDER_ID_1, STR_VALUE_1)
     val dataProvider = dataProviders.combineAsync<String, String, String>(
-      COMBINED_PROVIDER_ID, baseProvider1, baseProvider2) { _, _ ->
+      COMBINED_PROVIDER_ID, baseProvider1, baseProvider2
```

Comment:
@vinitamurthi Here I think the earlier format made more sense to me.

Reply:
Oh? Hmm @anandwana001 do you know what the new ktlint rule is that is enforcing this new line one? We may have to disable it individually

Reply:
is it `no-blank-line-before-rbrace`

Reply:
Yes that seems to be the one. But @rt4914 from https://github.com/oppia/oppia-android/pull/1234/files#r434257078 it seems like we do want these in new lines. I dont think ktlint behaves differently for `}` or ')'
Given that, shouldn't we keep this?

Reply:
Okay, that makes sense. Then in that case we cannot change this. Got it. thanks.